client/src/main/java/org/apache/cassandra/sidecar/client/SidecarClient.java

```diff
@@ -517,10 +518,11 @@ public CompletableFuture<Void> updateRestoreJob(String keyspace,
      * {@inheritDoc}
      */
     @Override
-    public CompletableFuture<Void> abortRestoreJob(String keyspace, String table, UUID jobId)
+    public CompletableFuture<Void> abortRestoreJob(String keyspace, String table, UUID jobId,
```

Comment:
I feel we should preserve the existing signature and add another method that takes payload. For the existing signature, the payload would be null.